explorer/app/apis/catalog/ncpi-catalog/common/entities.ts

```diff
@@ -1,40 +1,40 @@
-/**
- * Model of NCPI catalog.
- */
-export interface NCPICatalog {
-  consentCodes: string[];
-  dataTypes: string[];
-  dbGapId: string;
-  focusDisease: string;
-  participantCount: number;
-  platforms: string[];
-  study: string;
-  studyAccession: string;
-  studyDesigns: string[];
-}
+// /**
+//  * Model of NCPI catalog.
+//  */
+// export interface NCPICatalog {
+//   consentCodes: string[];
+//   dataTypes: string[];
+//   dbGapId: string;
+//   focusDisease: string;
+//   participantCount: number;
+//   platforms: string[];
+//   studyAccession: string;
+//   studyDesigns: string[];
+//   title: string;
+// }
 
 export type NCPICatalogEntity = NCPICatalogPlatform | NCPICatalogStudy;
 
 export interface NCPICatalogPlatform {
   consentCodes: string[];
   dataTypes: string[];
-  dbGapId: string[]; // dbGapIds - a list of study ids.
-  focusDisease: string[]; // focusDiseases - a list of focuses / diseases.
+  dbGapIds: string[]; // dbGapIds - a list of study ids.
```

Comment:
Revert 21 - 27 so we can filter terms across tabs.

Reply:
OK I did a bit of cleanup for NCPI-catalog here. I just realized I did not update Anvil-catalog for this. So stay tune on this one.